packages/dashmate/src/listr/tasks/setup/regular/importCoreDataTaskFactory.js

```diff
@@ -0,0 +1,221 @@
+          });
+
+          // Read configuration from dashd.conf
+          const configPath = path.join(coreDataPath, 'dash.conf');
+          const configFileContent = fs.readFileSync(configPath, 'utf8');
+          const masternodeOperatorPrivateKey = configFileContent.match(/^masternodeblsprivkey=([^ \n]+)/m)[1];
+
+          if (masternodeOperatorPrivateKey) {
+            ctx.config.set('core.masternode.operator.privateKey', masternodeOperatorPrivateKey);
+          }
+
+          const host = configFileContent.match(/^bind=([^ \n]+)/m)[1];
+
+          if (host) {
+            ctx.config.set('core.p2p.host', host);
+          }
+
+          // Store values to fill in the configure node form
+
+          // eslint-disable-next-line prefer-destructuring
+          ctx.importedP2pPort = configFileContent.match(/^port=([^ \n]+)/m)[1];
+
+          // eslint-disable-next-line prefer-destructuring
+          ctx.importedExternalIp = configFileContent.match(/^externalip=([^ \n]+)/m)[1];
+
+          // Copy data directory to docker a volume
+
+          // Create a volume
+          const { COMPOSE_PROJECT_NAME: composeProjectName } = generateEnvs(ctx.config);
+
+          const volumeName = `${composeProjectName}_core_data`;
+
+          // Check if volume already exists
+          const volumes = await docker.listVolumes();
+          const exists = volumes.Volumes.some((volume) => volume.Name === volumeName);
+
+          if (exists) {
+            throw new Error(`Volume ${volumeName} already exists. Please remove it first.`);
+          }
+
+          await docker.createVolume(volumeName);
+
+          // Pull image
+          await dockerPull('alpine');
+
+          const commands = [
+            `mkdir /${volumeName}/.dashcore/`,
+            'cd /source',
+            `cp -avL * /${volumeName}/.dashcore/`,
+            `chown -R 1000:1000 /${volumeName}/`,
+            `rm /${volumeName}/.dashcore/dash.conf`,
+          ];
+
+          // Copy data and set user dash
+          const [result] = await docker.run(
+            'alpine',
+            [
+              '/bin/sh',
+              '-c',
+              commands.join(' && '),
+            ],
+            task.stdout(),
+            {
+              HostConfig: {
+                AutoRemove: true,
+                Binds: [
+                  `${coreDataPath}:/source:ro`,
+                ],
+                Mounts: [
+                  {
+                    Type: 'volume',
+                    Source: volumeName,
+                    Target: `/${volumeName}`,
+                  },
+                ],
+              },
+            },
+          );
+
+          if (result.StatusCode !== 0) {
+            throw new Error('Cannot copy data dir to volume');
+          }
+
+          // TODO: Wording needs to be updated
+          await task.prompt({
+            type: 'confirm',
+            header: `  You need to stop your existing node before your start a dashmate
+    node\n`,
```

Comment:
Maybe we should also mention disabling any auto-start cron/script for the existing Core-only install too. That could lead to conflicts on reboots.
```suggestion
            header: `  Please stop your existing Core node before starting the new dashmate-based
    node\n`,
```

Reply:
@shumkov updated per discussion